src/rdkafka_lz4.c

```diff
@@ -210,13 +210,15 @@ rd_kafka_lz4_decompress (rd_kafka_broker_t *rkb, int proper_hc, int64_t Offset,
         }
 
         /* If uncompressed size is unknown or out of bounds, use a sane
-         * default (2x compression) and reallocate if needed
-         * More info on max size: http://stackoverflow.com/a/25751871/1821055 */
+         * default (4x compression) and reallocate if needed
+         * More info on max size: http://stackoverflow.com/a/25751871/1821055 
+         * More info on lz4 compression ratios seen for different data sets:
+         * http://dev.ti.com/tirex/content/simplelink_msp432p4_sdk_1_50_00_12/docs/lz4/users_guide/docguide.llQpgm/benchmarking.html
```

Comment:
That link suggests something closer to 2x as the "default" rate, but 4 is fine and is alot better than 255 :)